lib/app/views/list_my_friend/list_my_friend_widget.dart

```diff
@@ -0,0 +1,313 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:hit_moments/app/core/constants/assets.dart';
+import 'package:hit_moments/app/core/constants/color_constants.dart';
+
+import '../../models/friend.model.dart';
+import '../../models/user_model.dart';
+import 'components/list_friend_proposal.dart';
+import 'components/list_my_friend.dart';
+
+class ListMyFriendWidget extends StatefulWidget {
+  const ListMyFriendWidget(
+      {super.key, required this.friend, required this.users});
+
+  final Friend friend;
+  final List<User> users;
+
+  @override
+  State<ListMyFriendWidget> createState() => _ListMyFriendWidgetState();
+}
+
+class _ListMyFriendWidgetState extends State<ListMyFriendWidget> {
+  bool checkColorList = false;
+  bool isExpandedMyFriend = false;
+  bool isExpandedFriendSuggestions = false;
+  bool isDownUpMyFriend = false;
+  bool isDownUpFriendSuggestions = false;
+
+  void toggleColor(String title) {
+    setState(() {
+      if (title == AppLocalizations.of(context)!.list) {
```

Comment:
Cái này dùng S.of(context) này, vô file l10n.dart xem, hôm đấy đã code cho 1 cái đễ đỡ dài rồi còn...